exec/api/src/main/java/org/jboss/windup/exec/configuration/options/OutputPathOption.java

```diff
@@ -77,17 +77,19 @@ public static ValidationResult validateInputAndOutputPath(Path inputPath, Path o
         return validateInputsAndOutputPaths(Collections.singletonList(inputPath), outputPath);
     }
 
+    @SuppressWarnings("rawtypes")
     public static ValidationResult validateInputsAndOutputPaths(Collection inputPaths, Path outputPath)
     {
-        if (outputPath == null)
-        {
-            return new ValidationResult(ValidationResult.Level.ERROR, "Output path must be specified.");
-        }
 
         if (inputPaths == null || inputPaths.isEmpty())
         {
             return new ValidationResult(ValidationResult.Level.ERROR, "Input path must be specified.");
         }
+        
+        if (outputPath == null)
+        {
+            return new ValidationResult(ValidationResult.Level.ERROR, "Output path must be specified.");
+        }
 
```

Comment:
Switching the order of check, `inputPath` is more important than `outputPath`

Reply:
ok

Reply:
Although in this case, it doesn't really matter because this is the API part, right? I.e. dev time error.
If it's not, then what's the handling of default at the top of this PR?

Reply:
Not really sure, what you are trying to say by that "API part".